src/llfs/storage_file_builder.test.cpp

```diff
@@ -81,29 +81,26 @@ class StorageFileBuilderTest : public ::testing::Test
       for (const llfs::PackedConfigSlot& slot : p_config_block.slots) {
         // Verify PackedPageDeviceConfig
         //
-        BATT_ASSERT_EQ(slot.tag, llfs::PackedConfigSlotBase::Tag::kPageDevice);
+        ASSERT_EQ(slot.tag, llfs::PackedConfigSlotBase::Tag::kPageDevice);
         const llfs::PackedPageDeviceConfig& page_device_config =
             llfs::config_slot_cast<llfs::PackedPageDeviceConfig>(slot);
-        if (!(page_device_config.page_0_offset == expected_page0_offset &&
-              page_device_config.device_id == expected_device_id &&
-              page_device_config.page_count == kTestPageCount &&
-              page_device_config.page_size_log2 == page_size_log2)) {
-          return false;
-        }
+        ASSERT_EQ(page_device_config.page_0_offset, expected_page0_offset);
+        ASSERT_EQ(page_device_config.device_id, expected_device_id);
+        ASSERT_EQ(page_device_config.page_count, (little_i64)kTestPageCount);
+        ASSERT_EQ(page_device_config.page_size_log2, page_size_log2);
+
         // Update page offset numbers and device ID
         //
         expected_page0_offset +=
             page_device_config.page_size() * kTestPageCount - llfs::PackedConfigSlot::kSize;
         expected_device_id += 1;
       }
 
-      // Set expected slots filled in config block. Full config block contains 62 slots
+      // Assuming 1 slot filled if back slot, all slots filled otherwise.
       //
-      if (expected_device_id % 62 == 0) {
-        expected_num_slots = 62;
-      } else {
-        expected_num_slots = expected_device_id % 62;
-      }
+      constexpr u64 kMaxPageDeviceSlots =
```

Comment:
This value would be great to have as a member of PackedConfigBlock, I think.  It should be called something more generic than `kMaxPageDeviceSlots` since the slots can have many types; maybe `PackedConfigBlock::kMaxSlots`?

Reply:
I kept this name specific because the size of the slot headers can be different, kSizeof PackedVolumeConfig is "static constexpr usize kSize = PackedConfigSlot::kSize * 2;"

It would be easy to add to PackedConfigBlock, but since slot headers could be different sizes, would that be appropriate?

All of the PackedConfigHeaderSizes kSize are derived from PackedConfigSlot::kSize except for PackedPageArenaConfig, so I will make this change to more inline with the others.

Reply:
Added PackedConfigBlock::kMaxSlots.

In regard to the other point, see change to page_arena_config.hpp. PackedPageArenaConfig::kSize was previously defined as 64 instead of from PackedConfigSlot::kSize, like PackedPageAllocatorConfig::kSize, PackedVolumeConfig::kSize, ect... were defined.